examples/play_buffer.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python3
+
+import argparse
+import ctypes as ct
+import os
+
+import vlc
+
```

Comment:
It would help to have some docstring here, explaining the overall goal of the example and describing in simple sentences how it works, and the APIs it uses.